buildozer/__init__.py

```diff
@@ -935,8 +935,7 @@ def usage(self):
 
             if not meth.__doc__:
                 continue
-            doc = [x for x in
-                    meth.__doc__.strip().splitlines()][0].strip()
+            doc = list(meth.__doc__.strip().splitlines())[0].strip()
```

Comment:
IIRC, splitlines() is already returning a list - so why has the deepsource bot not removed the redundant `list()` constructor?

Reply:
It is unnecessary to use a comprehension just to loop over the iterable and create a list/set/dict out of it. Python has a specialized set of tools for this task: the list/set/dict constructors, which are faster and more readable.

Reply:
@AndiEcker the suggestion is correct and thanks for pointing it out.